public/models/Img.js

```diff
@@ -32,34 +32,34 @@ class Img {
     this.size = size || 1;
   }
 
-    //Marianna
-    //draw and change animation based on data in spritesheet
-    draw (ctx, x, y, width, height) {
-        //where on the spritesheet should we start - x and y coordinates
-        //console.log(this.currentSpeed, this.speed);
-        let startX = this.currentColumn * width;
-        let startY = this.currentRow * height;
-        //increase the current speed value
-        this.currentSpeed++;
-        //if current speed value equals defined speed, then we change the animation frame
-        if (this.currentSpeed === this.speed) {
-            //increase the column - our animations go from left to right and then to the next line so firstly we increase columns
-            this.currentColumn++;
-            //reset currentSpeed to 0 again to be prepared for next call of the function
-            this.currentSpeed = 0;
-        }
-        //if the value of currentColumn is higher than the last column, we reset it to 1st one and move to the next row
-        if (this.currentColumn > this.startColumn + this.columns) {
-            this.currentColumn = this.startColumn;
-            this.currentRow++;
-            //if current row is higher than the last row, we reset it to the first one
-            if (this.currentRow > this.startRow + this.rows){
-                this.currentRow = this.startRow;
-            }
-        }
-        //draw the image (spritesheet IMG, start x on img, start y on img, width, height, x on canvas, y on canvas, width on canvas, height on canvas)
-        ctx.drawImage(this.img, startX, startY, width, height, x, y, width * this.size, height * this.size);
+  // Marianna
+  // draw and change animation based on data in spritesheet
+  draw(ctx, x, y, width, height) {
+    // where on the spritesheet should we start - x and y coordinates
+    // console.log(this.currentSpeed, this.speed);
+    const startX = this.currentColumn * width;
+    const startY = this.currentRow * height;
+    // increase the current speed value
+    this.currentSpeed++;
```

Comment:
why are increasing speed?

Reply:
That's a speed of animation. As each animation has few pictures that is is changing between. Speed probably isn't the best name for this but I'm not sure how else to call it. Basically for every time draw method is called it increases this value, and when it reaches the defined value (which for now im mostly using 5) aka draw method is called 5 times, then it changes the animation picture.